public/index.html

```diff
@@ -19,6 +19,51 @@
   </head>
   <body>
     <noscript>You need to enable JavaScript to run this app.</noscript>
+    <script>
+      window.addEventListener(
+          'message',
+          ({ origin, data: { action, requestId, store } }) => {
+            if (origin !== 'https://editor.unlayer.com') {
+              return;
+            }
+
+            if (action !== 'getPromoCodes') {
+              return;
+            }
+
+            window.frames[0].postMessage(
+              {
+                isResponse: true,
+                requestId: requestId,
+                value: [
+                  {
+                    code: 'abc-1',
+                    type: 'money',
+                    value: 1000,
+                    useLimit: 1,
+                    minPaymentAmount: 1,
+                  },
+                  {
+                    code: 'cde-2',
+                    type: 'money',
+                    value: 1500,
+                    useLimit: 1,
+                    minPaymentAmount: 1,
+                  },
+                  {
+                    code: 'efg-3',
+                    type: 'percen',
+                    value: 15,
+                    useLimit: 1,
+                    minPaymentAmount: 1,
+                  },
+                ],
+              },
+              { targetOrigin: '*' },
+            );
+          },
```

Comment:
What do you think about moving this code to [App.tsx](https://github.com/FromDoppler/unlayer-editor/blob/main/src/App.tsx)?

Reply:
> let me see how to add it in the "React way".

Oh, you are right, in that case, probably adding it to [index.tsx](https://github.com/FromDoppler/unlayer-editor/blob/main/src/index.tsx) as plain javascript could be a better option

Reply:
In Editors WebApp, we are not using the "React way": https://github.com/FromDoppler/doppler-editors-webapp/pull/661/files#diff-0b5adbfe7b36e4ae2f479291e20152e33e940f7f265162d77f40f6bdb5da7405R23-R24

Reply:
done.